features/fixed_costs_form.feature

```diff
@@ -3,14 +3,20 @@ Feature: Be able to enter monthly fixed costs
   In order to know what my fixed cost total is
   I should be able to enter all the fields
 
-Scenario: Fills in fixed costs form
-  Given I am on home page
-  When I click 'New Expense'
-  And I should see 'Enter Fixed Expenses'
-  And I fill in 'Rent' field with '30000'
-  And I fill in 'Insurance' field with '300'
-  And I fill in 'Transportation' field with '1000'
-  And I fill in 'Education' field with '380'
-  And I fill in 'Food' field with '3500'
-  And I click 'Submit'
-  And I should see 'Expense saved!'
+  Background:
+    Given following user exist
+    | email                 | password |
+    | antonella@email.com   | password123 |
+    And I am logged in as 'antonella@email.com'
+
+  Scenario: Fills in fixed costs form
+    Given I am on home page
+    When I click 'New Expense'
+    And I should see 'Enter Fixed Expenses'
```

Comment:
Then

Reply:
Or are you saying just the first test should be what you comment is? we need a little bit more detail than that